src/frontend/apps/e2e/__tests__/app-desk/mail-domain.spec.ts

```diff
@@ -0,0 +1,350 @@
+      },
+      {
+        id: '3e7c8639-3a6d-4267-918e-69c7284beeb9',
+        local_part: 'snobour2',
+        secondary_email: 'sebastien.nobour2@gmail.com',
+      },
+      {
+        id: '493fb661-042d-4ffe-ad7b-ad4b072f133b',
+        local_part: 'snobour3',
+        secondary_email: 'sebastien.nobour3@gmail.com',
+      },
+      {
+        id: '43ad7e76-3070-4b71-88ab-036a56208714',
+        local_part: 'snobour4',
+        secondary_email: 'sebastien.nobour4@gmail.com',
+      },
+      {
+        id: 'def7c046-c4cc-4985-87ff-452853466411',
+        local_part: 'snobour5',
+        secondary_email: 'sebastien.nobour5@gmail.com',
+      },
+      {
+        id: 'c057555b-9f02-421b-bfc8-6e07585d63ab',
+        local_part: 'snobour6',
+        secondary_email: 'sebastien.nobour6@gmail.com',
+      },
+      {
+        id: '939d6d0f-3ae2-4fdf-9bfa-345bd0b0b49c',
+        local_part: 'snobour7',
+        secondary_email: 'sebastien.nobour7@gmail.com',
+      },
+      {
+        id: '4a21e507-7cbd-48ad-a322-f0b3459e3607',
+        local_part: 'snobour8',
+        secondary_email: 'sebastien.nobour8@gmail.com',
+      },
+      {
+        id: '4e84b5a3-07bc-433b-a1a3-f2b8367d98a3',
+        local_part: 'snobour9',
+        secondary_email: 'sebastien.nobour9@gmail.com',
+      },
+      {
+        id: '0e49064f-3b57-4154-9771-30cb8d948b2e',
+        local_part: 'snobour10',
+        secondary_email: 'sebastien.nobour10@gmail.com',
+      },
+      {
+        id: 'b4e28e6a-9704-4c6e-9a63-9665ddac427e',
+        local_part: 'snobour11',
+        secondary_email: 'sebastien.nobour11@gmail.com',
+      },
+      {
+        id: 'ed77fce9-a489-413f-9479-a6367c00685a',
+        local_part: 'snobour12',
+        secondary_email: 'sebastien.nobour12@gmail.com',
+      },
+      {
+        id: '729685c0-22cd-4347-822a-b857a63c3fdd',
+        local_part: 'snobour13',
+        secondary_email: 'sebastien.nobour13@gmail.com',
+      },
+      {
+        id: '5c51f9c7-1675-4c28-aaf8-e0221bfc451b',
+        local_part: 'snobour14',
+        secondary_email: 'sebastien.nobour14@gmail.com',
+      },
+      {
+        id: '00e871d6-975b-4564-9055-54c858c82dc1',
+        local_part: 'snobour15',
+        secondary_email: 'sebastien.nobour15@gmail.com',
+      },
+      {
+        id: '749a550f-a172-43be-9726-3ed162ddaf46',
+        local_part: 'snobour16',
+        secondary_email: 'sebastien.nobour16@gmail.com',
+      },
+      {
+        id: 'dbe23e9c-33c3-4435-913b-bd17d03f3628',
+        local_part: 'snobour17',
+        secondary_email: 'sebastien.nobour17@gmail.com',
+      },
+      {
+        id: '13595199-4fa4-4d41-bf42-ea94685872c0',
+        local_part: 'snobour18',
+        secondary_email: 'sebastien.nobour17@gmail.com',
+      },
+      {
+        id: 'baa2d4cc-a167-4130-9047-899ef3b1792a',
+        local_part: 'snobour19',
+        secondary_email: 'sebastien.nobour19@gmail.com',
+      },
```

Comment:
I think you can put less dummy data, the test will react the same way with less data and it increases readibility.

```suggestion
```

Reply:
I left this array the way it was as I was testing the pagination and that automating the generation of these 20 fixtures would take longer.